buildpacks/nodejs-function-invoker/lib/failures.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+fail_on_no_main_file() {
+	build_dir=$1
+	fn_entry_file=$(json_get_key "$build_dir/package.json" ".main")
```

Comment:
Should this also handle the case where the `main` key doesn't exist at all in `package.json`?

Reply:
Yeah good call, I'll create another failure method for this.